pkgs/applications/networking/instant-messengers/salut-a-toi/default.nix

```diff
@@ -1,7 +1,12 @@
-{stdenv, fetchurl, python27Packages, file }:
+{ stdenv, fetchurl, python27Packages, file, pkgs }:
```

Comment:
Don't use `pypi2nix` in 5666dff703372a9aaa0ec5fe03a2edd713f9f4ee.

Reply:
I did not use pypi2nix for any of this.

Reply:
Right, I see. In any case, functions should preferably have exact arguments instead of package sets.